kaldb/src/main/java/com/slack/kaldb/metadata/schema/FieldTypeUtils.java

```diff
@@ -0,0 +1,10 @@
+package com.slack.kaldb.metadata.schema;
+
+import org.apache.lucene.analysis.core.KeywordAnalyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+
+public class FieldTypeUtils {
```

Comment:
Not sure if we need this separate class just yet - if we do though, we should have comments explaining the purpose of this "utils" class, since this is a good chance for it to become a catch-all.